packages/sanity/src/core/versions/components/BundleMenu.tsx

```diff
@@ -0,0 +1,126 @@
+import {CheckmarkIcon} from '@sanity/icons'
+import {Box, Flex, Menu, MenuButton, MenuDivider, MenuItem, Spinner, Text} from '@sanity/ui'
+import {type ReactElement, useCallback, useContext} from 'react'
+import {RelativeTime} from 'sanity'
+import {styled} from 'styled-components'
+
+import {
+  VersionContext,
+  type VersionContextValue,
+} from '../../../_singletons/core/form/VersionContext'
+import {type BundleDocument} from '../../store/bundles/types'
+import {LATEST} from '../util/const'
+import {isDraftOrPublished} from '../util/dummyGetters'
+import {BundleBadge} from './BundleBadge'
+
+const StyledMenu = styled(Menu)`
+  min-width: 200px;
+`
+
+interface BundleListProps {
+  button: ReactElement
+  bundles: BundleDocument[] | null
+  loading: boolean
+  actions?: ReactElement
+}
+
+export function BundleMenu(props: BundleListProps): JSX.Element {
+  const {bundles, loading, actions, button} = props
+
+  // eslint-disable-next-line no-warning-comments
```

Comment:
COULD: add these eslint rules to the `.eslintrc` global, where we also turn off the string literal rules for releases

An aside, being that we might also want to change [this](https://github.com/sanity-io/sanity/blob/corel/.eslintrc.cjs#L166) to:

`files: ['**/*/core/releases/**/*', '**/*/core/version/**/*']`

Reply:
I feel this would go against the point of why I'm adding them on the files:
I want them to be intrusive and very visible these things NEED to be discussed before we push it to `next` and a reviewer should raise them up if we end up having them in the PR for review.

If we were to add that, then I think it would beat the point. Normal comments, imo, are often brushed over in reviews, linting disables raise more of a brow 😄